op-chain-ops/cmd/op-upgrade/main.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/ethereum-optimism/optimism/op-chain-ops/safe"
 	"github.com/ethereum-optimism/optimism/op-chain-ops/upgrades"
 
+	op_node_genesis "github.com/ethereum-optimism/optimism/op-node/cmd/genesis"
```

Comment:
nit: very non standard to import things from `cmd` directory. `cmd` doesn't contain libraries, it should contain the minimal stuff to make an executable. You should move `WriteJSON` to the json package in `op-service`

Reply:
addressed. thank you!